apps/builder/app/builder/features/inspector/inspector.tsx

```diff
@@ -51,7 +50,15 @@ export const Inspector = ({ publish, navigatorLayout }: InspectorProps) => {
     }
   }, [selectedInstance, tab]);
 
-  if (navigatorLayout === "docked" && dragAndDropState.isDragging) {
+  useEffect(() => {
+    return dragAndDropStateContainer.subscribe((dragAndDropState) => {
+      setShowNavigatorTree(
+        navigatorLayout === "docked" && dragAndDropState.isDragging
+      );
+    });
+  }, [navigatorLayout]);
+
+  if (showNavigatorTree) {
```

Comment:
Or even better. Add computed to access only isDragging

Reply:
Does it don't rerender now?

Reply:
Trying, thank you